### PR TITLE
#34 add CreateFeedbackDTO

### PR DESCRIPTION
**Pull request by alesyalis**
Remove the CreateOn from FeedbackDTO at the input and create it through set map, leave dto at the output with CreateOn

### CODE REVIEW

File: Recrutify/Recrutify.Services/Services/CandidateService.cs [hunk 1]
```diff
@@ -40,10 +40,10 @@ public async Task<CandidateDTO> CreateAsync(CandidateCreateDTO candidateCreateDT
             return result;
         }
 
-        public Task UpsertAsync(Guid id, Guid projectId, FeedbackDTO feedbackDto)
+        public async Task UpsertFeedbackAsync(Guid id, Guid projectId, CreateFeedbackDTO feedbackDto)
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Services/CandidateService.cs:43]: Remove async and await and write just return

File: Recrutify/Recrutify.Services/DTOs/CandidateCreateDTO.cs [hunk 2]
```diff
@@ -27,8 +27,6 @@ public class CandidateCreateDTO
 
         public bool GoingToExadel { get; set; }
 
-        public List<ProjectResultDTO> ProjectResults { get; set; }
```
[the remark this answers is not in the document]
Comment (alesyalis) [reply Recrutify/Recrutify.Services/DTOs/CandidateCreateDTO.cs:30]: done

File: Recrutify/Recrutify.Services/DTOs/CreateFeedbackDTO.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        public string TextFeedback { get; set; }
+
+        public int Rating { get; set; }
+
+        public Guid UserId { get; set; }
+
+        public FeedbackTypeDTO Type { get; set; }
+
+        public bool IsRecommended { get; set; }
```
[the remark this answers is not in the document]
Comment (alesyalis) [reply Recrutify/Recrutify.Services/DTOs/CreateFeedbackDTO.cs:19]: done

File: Recrutify/Recrutify.Services/Services/CandidateService.cs (hunk 1 above)
Comment (alesyalis) [reply Recrutify/Recrutify.Services/Services/CandidateService.cs:43]: done

File: Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs [hunk 4]
```diff
@@ -10,10 +10,14 @@ public CandidateProfile()
         {
             CreateMap<Candidate, CandidateDTO>().ReverseMap();
             CreateMap<CandidateCreateDTO, Candidate>()
+                .ForMember(x => x.ProjectResults, opt => opt.Ignore())
+                .ForMember(x => x.RegistrationDate, opt => opt.Ignore())
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs:14]: Not ignore, init this field

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**